module1/homework/grayscaleImages/main.cpp

```diff
@@ -1,7 +1,7 @@
 #include <array>
 #include <forward_list>
 
-// TODO: include
+#include "compression.hpp"
```

Comment:
You can remove array include since it is included in the header.

Reply:
You are right, forgot about it.  Thank you.

Reply:
@WBigaj I use array in main.cpp so I include it. Doesn't matter it's included in compression.hpp also.

Reply:
But you do not need to include it one more time. By including header you include everything in it including all included libriaries.

Reply:
Sorry. My mistake i wasn't aware of "Include what you use rule".